services/horizon/internal/scripts/check_release_hash/Dockerfile

```
@@ -1,5 +1,5 @@
 # Change to Go version used in CI or rebuild with --build-arg.
-ARG GO_IMAGE=golang:1.14.15-stretch
+ARG GO_IMAGE=golang:1.16.2-stretch
```

```suggestion
ARG GO_IMAGE=golang:1.16.1-stretch
```
SDF's build server currently uses 1.16.1. To allow reproducible builds let's stick to 1.16.1.

---

Can we bump that to `.2`?

---

@bartekn SDF Horizon builds are now Dockerized so they don't use go from the buildserver. We just need to bump image in the build job to match this

---

We can bump it to `.2`. I'm just thinking that the next time we bump Golang version in our org we should create a meta issue with a list of places we need to update (right now I think it's Circle-CI, docker images, build server).

---

Resolving since we've updated to 1.16.1 now.